addons/l10n_lv/views/res_partner_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="view_partner_form">
+        <field name="name">res_partner.form.title</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+                <field name="title" position="after">

```

Comment:
what is intended here?
An l10n should avoid changing the partner form.

Reply:
It seems this file can be removed

Reply:
file removed